frame/contracts/src/wasm/mod.rs

```diff
@@ -1461,7 +1462,7 @@ mod tests {
 			vec![0x00, 0x01, 0x2a, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xe5, 0x14, 0x00])
 		]);
 
-		assert_eq!(gas_meter.gas_left(), 9834099446);
```

Comment:
wasn't the idea of exact numbers is that we don't commit something that can change gas cost and break consensus?

Reply:
I didn't know that. Given that only those two seemingly random tests had a check for an explicit gas costs I didn't thought there where some kind reason behind that. I would expected an explicit test exercising more than one functionality to check that the gas costs changes without changing the `Schedule`.